conf/application.conf

```diff
@@ -20,6 +20,8 @@ formMaxAttachmentSizeMB = 10
 
 restrictedFileExtensionList = ["jfif", "png"]
 
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientV2Module"
```

Comment:
This line does nothing, since gform is using it's own  custom `ApplicationLoader`, so it would be better to remove it, to avoid any future confusion.